src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -388,7 +388,7 @@ public function handleUnconditionally($value = null)
      *       return "http://stackoverflow.com";
      *   });
      * @param string $identifier
-     * @param string $resolver
+     * @param string|\Closure $resolver
```

Comment:
`callable`?

Reply:
Yes, I just read the method description now, sorry :sweat_smile: